web/templates/empty_list_widget_for_list.hbs

```diff
@@ -0,0 +1 @@
+<li id="empty-list-message">{{empty_list_message}}</li>
```

Comment:
Are we okay having `id`s that occur multiple times in the DOM? I thought it was best practice to use classnames for things we can't guarantee only appear once.

Reply:
@HemantUmre12 is there a reason for this and `empty-table-message` be not classes?

Reply:
Opened #27225

Reply:
Sorry for the late reply. I used `id` here to raise the specificity of the style. I agree with @evykassirer that if we can avoid using `id`, it would be better. Alternatively, we might need to write couple lines of CSS in a few places.